terraform/modules/neptune/main.tf

```diff
@@ -0,0 +1,77 @@
+resource "aws_neptune_cluster" "neptune_cluster" {
+  apply_immediately                    = true
+  backup_retention_period              = var.backup_retention_period
+  cluster_identifier                   = "${var.resource_prefix}-neptune-cluster"
+  deletion_protection                  = var.deletion_protection
+  enable_cloudwatch_logs_exports       = ["audit"]
+  engine                               = "neptune"
+  engine_version                       = var.neptune_engine_version
+  iam_roles                            = var.neptune_iam_roles
+  neptune_subnet_group_name            = var.create_neptune_subnet_group ? aws_neptune_subnet_group.this[0].name : var.db_subnet_group_name
+  port                                 = 8182
+  preferred_backup_window              = var.backup_window
+  preferred_maintenance_window         = var.maintenance_window #wed:04:00-wed:09:00"
+  skip_final_snapshot                  = var.skip_final_snapshot
+  storage_encrypted                    = var.storage_encrypted
```

Comment:
This should be set to true instead of being parameterized with variables.

Reply:
we should encrypt by default but I think this is the right way to do this - there may be cases where that is not needed/desired